Tools/Cli-LoRa-Device-Provisioning/Program.cs

```diff
@@ -0,0 +1,543 @@
+                var twinData = JObject.Parse(twin.Properties.Desired.ToJson());
+                twinData.Remove("$metadata");
+                twinData.Remove("$version");
+                Console.ForegroundColor = ConsoleColor.Yellow;
+                Console.WriteLine(twinData.ToString());
+                Console.ResetColor();
+            }
+
+            return isSuccess;
+        }
+
+        private static object RunUpdateAndReturnExitCode(UpdateOptions opts)
+        {
+            if (!configurationHelper.ReadConfig())
+                return false;
+
+            bool isSuccess = false;
+
+            opts = iotDeviceHelper.CleanOptions(opts as object, false) as UpdateOptions;
+            opts = iotDeviceHelper.CompleteMissingUpdateOptions(opts, configurationHelper);
+
+            var twin = iotDeviceHelper.QueryDeviceTwin(opts.DevEui, configurationHelper).Result;
+
+            if (twin != null)
+            {
+                twin = iotDeviceHelper.UpdateDeviceTwin(twin, opts);
+
+                if (iotDeviceHelper.VerifyDeviceTwin(opts.DevEui, opts.NetId, twin, configurationHelper))
+                {
+                    isSuccess = iotDeviceHelper.WriteDeviceTwin(twin, opts.DevEui, configurationHelper, false).Result;
+
+                    if (isSuccess)
+                    {
+                        Console.WriteLine($"Device {opts.DevEui} updated.");
+
+                        var newTwin = iotDeviceHelper.QueryDeviceTwin(opts.DevEui, configurationHelper).Result;
+                        var newTwinData = JObject.Parse(newTwin.Properties.Desired.ToJson());
+                        newTwinData.Remove("$metadata");
+                        newTwinData.Remove("$version");
+                        Console.ForegroundColor = ConsoleColor.Yellow;
+                        Console.WriteLine(newTwinData.ToString());
+                        Console.ResetColor();
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Can not update device {opts.DevEui}.");
+                    }
+                }
+                else
+                {
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine();
+                    Console.WriteLine($"Errors found in Twin data. Device {opts.DevEui} NOT updated.");
+                    Console.ResetColor();
+                }
+            }
+            else
+            {
+                Console.WriteLine($"Could not get data for device {opts.DevEui}. Failed to update.");
+                isSuccess = false;
+            }
+
+            return isSuccess;
+        }
+
+        private static object RunRemoveAndReturnExitCode(RemoveOptions opts)
+        {
+            if (!configurationHelper.ReadConfig())
+                return false;
+
+            bool isSuccess = iotDeviceHelper.RemoveDevice(opts.DevEui, configurationHelper).Result;
```

Comment:
you could directly return here: return iotDeviceHelper.RemoveDevice...

Reply:
done!